lib/translated-file.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+var path = require('path');
+var File = require('vinyl');
+
+function getLangPrefix(lang, defaultLang) {
+  if(lang === defaultLang) {
+    return '';
+  }
+  var dir = lang.toLowerCase().replace(/_/g, '-');
```

Comment:
isn't this going to result in urls like `templates.yola.com/pt-br` and `yola.com/cn-zh/support`?